src/lstring.c

```diff
@@ -46,6 +46,46 @@ lstr_t *str_init(const char *str)
     }
     pstr->size = lsize(str);
     pstr->buffer = (char *)malloc(pstr->size + 1);
-    lstr_cpy(pstr->buffer, str, 3, 4, 3);
+    lstr_cpy(pstr->buffer, str, 3, 4, 1);
     return pstr;
 }
+
+// Function to concatenate ch2 string to ch1 string
+
+lstr_t *Lstrcat(char *ch1, const char *ch2)
```

Comment:
Use const qualifiers for input of ch1 and ch2.

Reply:
done